File: app/src/main/java/com/mxt/anitrend/base/custom/view/container/NotificationCardView.java
package com.mxt.anitrend.base.custom.view.container;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

public class NotificationCardView extends CardViewBase {

    public NotificationCardView(@NonNull Context context) {
        super(context);
    }

    public NotificationCardView(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public NotificationCardView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void onInit() {
        applyStyle(0);
    }
}

File: app/src/main/java/com/mxt/anitrend/base/custom/view/container/NotificationCardView.kt
package com.mxt.anitrend.base.custom.view.container

import android.content.Context
import android.util.AttributeSet

class NotificationCardView
@JvmOverloads
constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0,
) : CardViewBase(context, attrs, defStyleAttr) {
    override fun onInit() {
        applyStyle(0)
    }
}
